module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -96,8 +96,12 @@ namespace module::input {
             this->config.buttons.debounceThreshold = config["buttons"]["debounce_threshold"].as<int>();
 
             // Foot down config
-            this->config.footDown.fromLoad           = config["foot_down"]["from_load"].as<bool>();
-            this->config.footDown.certaintyThreshold = config["foot_down"]["certainty_threshold"].as<float>();
+            const std::string method = config["foot_down"]["method"].as<std::string>();
+            std::map<std::string, float> thresholds;
```

Comment:
have you considered making this an enum, instead of a std::string?

Reply:
I have not. This is used a string later on as well (in some graph emits)